### PR TITLE
remove simupop from performance figures

### PR DESCRIPTION
**Pull request by molpopgen**
update figures to remove simupop

### CODE REVIEW

File: sims/plot_RAM.py [hunk 1]
```diff
@@ -99,14 +99,12 @@
...
-ax_simupop_arg.set_title("simuPOP with pedigree tracking", fontsize='medium')
-ax_simupop.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
-ax_simupop_arg.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
-ax_simupop_arg.set_xticks([1e3, 1e4, 1e5])
-for ax in (ax_simupop, ax_simupop_arg):
+ax_fwdpp.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+ax_fwdpp_arg.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+ax_fwdpp_arg.set_xticks([1e3, 1e4, 1e5])
+ax_fwdpp.set_xticks([1e3, 1e4, 1e5])
+for ax in (ax_fwdpp, ax_fwdpp_arg):
```
Comment (ashander) [inline sims/plot_RAM.py]: revert?

File: sims/quick_plot.py [hunk 2]
```diff
@@ -134,14 +141,15 @@
...
-ax_simupop_arg.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
-ax_simupop_arg.set_xticks([1e3, 1e4, 1e5])
-for ax in (ax_simupop, ax_simupop_arg):
+#ax_simupop.set_title("simuPOP with neutral mutations", fontsize='medium')
+#ax_simupop_arg.set_title("simuPOP with pedigree tracking", fontsize='medium')
+#ax_simupop.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+#ax_simupop_arg.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+#ax_simupop_arg.set_xticks([1e3, 1e4, 1e5])
+for ax in (ax_fwdpp, ax_fwdpp_arg):
+    ax.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
```
Comment (ashander) [inline sims/quick_plot.py]: revert?

File: sims/quick_plot.py [hunk 3]
```diff
@@ -171,12 +180,11 @@
...
 
-for ax in (ax_fwdpp, ax_simupop):
+for ax in (ax_fwdpp,):
     ax.set_ylabel("Speedup due to\npedigree recording")
 ax_fwdpp.set_title("fwdpy11", fontsize='medium')
-ax_simupop.set_title("simuPOP", fontsize='medium')
-ax_simupop.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
-ax_simupop.set_xscale('log')
+ax_fwdpp.set_xlabel('Scaled recombination rate (' + r'$\rho = 4Nr$)')
+ax_fwdpp.set_xscale('log')
```
Comment (ashander) [inline sims/quick_plot.py]: revert?

File: sims/plot_RAM.py [hunk 4]
```diff
@@ -57,23 +58,21 @@
 ax_fwdpp.legend(loc='best',frameon=False)
 ax_fwdpp.set_title("fwdpy11 with neutral mutations", fontsize='medium')
 ax_fwdpp_arg.set_title("fwdpy11 with pedigree tracking", fontsize='medium')
```
Comment (ashander) [inline sims/plot_RAM.py]: in the above lines
s/fwdpy11 with //

File: sims/quick_plot.py [hunk 5]
```diff
@@ -57,15 +60,19 @@
 ax_fwdpp.legend(loc='upper left',frameon=False)
 ax_fwdpp.set_title("fwdpy11 with neutral mutations", fontsize='medium')
 ax_fwdpp_arg.set_title("fwdpy11 with pedigree tracking", fontsize='medium')
```
Comment (ashander) [inline sims/quick_plot.py]: in the above lines
s/fwdpy11 with //

File: sims/quick_plot.py [hunk 6]
```diff
@@ -78,37 +85,36 @@
...
     popsize = int(name[1])
     ax.plot(group.rho, group.speedup, marker=m,
             ms=4, linestyle=lstyle, label=r'$N = {:.0e}$'.format(popsize),
             color=colors[name[1]],
             markerfacecolor=mfacecolor)
 
-for ax in (ax_fwdpp, ax_simupop):
+for ax in (ax_fwdpp,):
     ax.set_ylabel("Speedup due to\npedigree recording")
 ax_fwdpp.set_title("fwdpy11", fontsize='medium')
```
Comment (ashander) [inline sims/quick_plot.py]: delete this line?

**Review by ashander — CHANGES_REQUESTED** [review]
Looks great, quick work.

But: I think we wanted to keep simuPOP in the figs related to neutral. I comment on the relevant lines below but the figs are

- memuse_nosel
- rawspeed_nosel
- speedup_nosel

Also we may want to remove the (possibly confusing) facet labels on top of the plots referencing fwdpy11 which now have no comparison. I commented on those too

**Comment by ashander** [comment]
Oh...  sorry requesting changes leaves that ugly red X

**Comment by ashander** [comment]
I realize the earlier email  wasn't super clear on retaining some of the simuPOP result. That was the intention of:

> 3. We would put current results in the appendix in place of the 'neutral' results, and could still refer to them, semi-tangentially.

File: sims/quick_plot.py (hunk 6 above)
Comment (molpopgen) [reply sims/quick_plot.py]: w/o it there is no y label

File: sims/quick_plot.py (hunk 2 above)
Comment (molpopgen) [reply sims/quick_plot.py]: w/o it there is no label

**Comment by molpopgen** [comment]
> But: I think we wanted to keep simuPOP in the figs related to neutral. I comment on the relevant lines below but the figs are

As I understood, plan was to dump simupop.  If we keep neutral, gotta rerun the selected...

File: sims/quick_plot.py (hunk 6 above)
Comment (ashander) [reply sims/quick_plot.py]: I mean line 106
`ax_fwdpp.set_title("fwdpy11", fontsize='medium')` which just removes the title, I think? Anyway that's what I meant

File: sims/quick_plot.py (hunk 2 above)
Comment (ashander) [reply sims/quick_plot.py]: just meant revert the is part of the diff, lines 144-152

Comment (molpopgen) [reply sims/quick_plot.py]: That is not at all what GH is showing me in the browser.

**Comment by molpopgen** [comment]
> We would put current results in the appendix in place of the 'neutral' results, and could still refer to them, semi-tangentially.

But we, or at least I, no longer know what those results represent.

**Comment by ashander** [comment]
yes. good point and I agree. I don't agree either with the plain language of 

> We would put current results in the appendix in place of the 'neutral' results, and could still refer to them, semi-tangentially.

regarding current results in the main text. but think retaining ~neutral~ without selection results is worth it

**Comment by molpopgen** [comment]
Did a forced update, as the changes weren't really revertible before.

**Comment by petrelharp** [comment]
what is the conclusion: leave simupop in appendix or not?  "leave in" maybe the easiest?

**Review by ashander — APPROVED** [review]

**Comment by ashander** [comment]
How about: 

I'll do the simupop edit, with "leave it in" using these figs. If we decide that's bad, @molpopgen 's earlier work to remove it completely is preserved here https://github.com/petrelharp/ftprime_ms/tree/remove-simupop-fully

**Comment by petrelharp** [comment]
ok great.

**Comment by ashander** [comment]
oops. now this 

>  @molpopgen 's earlier work to remove it completely is preserved here https://github.com/petrelharp/ftprime_ms/tree/remove-simupop-fully

is actually true praise the reflog

**Pull request merged**